File: examples/node/send.ts
import { ArkeselClient } from '@kalourmade/arkesel-sms';

const client = new ArkeselClient(process.env.ARKESEL_API_KEY!);

const response = await client.sms.send({
  sender: 'Arkesel',
  recipients: ['233544919953'],
  message: 'Hello from the Arkesel Node SDK',
});

for (const result of response.results) {
  console.log(`Sent to ${result.recipient} — id=${result.id}`);
}
for (const invalid of response.invalidNumbers) {
  console.log(`Invalid number: ${invalid}`);
}

File: examples/python/send.py
import os

from kalourmade_arkesel import ArkeselClient

client = ArkeselClient(os.environ["ARKESEL_API_KEY"])

response = client.sms.send(sender="Arkesel", recipients=["233544919953"], message="Hello from the Arkesel Python SDK")

for result in response.results:
    print(f"Sent to {result.recipient} — id={result.id}")
for invalid in response.invalid_numbers:
    print(f"Invalid number: {invalid}")
